src/Models/Dto/UserDto.cs

```diff
@@ -12,17 +12,24 @@ public class UserDto : Dto
     [RedisIdField]
     public override string Id { get; set; } = default!;
     [Indexed]
+    public string UserName { get; set; }
```

Comment:
@radu-popescu, could you please add `= default!;` in the end, it will get rid of a warning :)

Reply:
resolved